cmd/idp-ldap-accesskey-create.go

```diff
@@ -143,6 +143,7 @@ func accessKeyCreateOpts(ctx *cli.Context, targetUser string) madmin.AddServiceA
 	}
 
 	var exp time.Time
+	var expPointer *time.Time
```

Comment:
@taran-p can we avoid adding a new local variable here and change 'exp' type to `* time.Time` ? it will be easier that way.

Reply:
I'm not sure how to do this without adding a different way to avoid null reference issues

Reply:
can you elaborate more about the null reference issue ?

Reply:
If exp is declared as a pointer, setting its value directly will be a null dereference and fault, so using an additional variable avoids this issue.